common/src/test/java/com/revenuecat/purchases/common/HTTPClientVerificationTest.kt

```diff
@@ -249,6 +250,30 @@ class HTTPClientVerificationTest: BaseHTTPClientTest() {
         assertThat(result.verificationResult).isEqualTo(VerificationResult.FAILED)
     }
 
+    @Test
+    fun `performRequest on informational client without nonce does not throw on verification error`() {
```

Comment:
I think that's an extra word:
```suggestion
    fun `performRequest on informational client without nonce does not throw verification error`() {
```